ecosystem/web-wallet/src/background.ts

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+import { AptosAccount, AptosClient, Types } from 'aptos'
+import { devnetNodeUrl } from './constants'
+import { getAptosAccountState } from './utils/account'
+
+chrome.runtime.onMessageExternal.addListener(async function (request, _sender, sendResponse) {
+  const account = getAptosAccountState()
+  if (account === undefined) {
+    sendResponse({ error: 'No Accounts' })
+  } else if (request.method === 'getAccountAddress') {
+    if (account.address()) {
```

Comment:
I would extract this block to a `getAccountAddress()` function. This will make the code easier to read as we add more message types.